userguide/src/main/jbake/content/config.inc

```diff
@@ -59,6 +58,12 @@ Configuring Your Environment to Run the TCK Against the Compatible Implementatio
       in {TechnologyShortName} {TechnologyVersion} Compatible Implementation.
 See link:#GCLHUC1[Section, "Configuration Properties"] for details.
 
+Note: Jakarta XML Binding TCK searches Compatible Implementation via Java SPI
+(Java Service Provider Interface). Implementation must contain
+`META-INF/services/jakarta.xml.bind.JAXBContext` and
+`META-INF/services/jakarta.xml.bind.JAXBContextFactory` with reference/content to
+implementation classes.
+
```

Comment:
the spec/api itself defines the search algorithm in https://javadoc.io/doc/jakarta.xml.bind/jakarta.xml.bind-api/latest/jakarta.xml.bind/jakarta/xml/bind/JAXBContext.html, the note should refer to that instead probably

Reply:
It seems, that there is problem with JavaTest Harness and test design itself.

-  `jaxb.properties` file is not used or generated except https://github.com/eclipse-ee4j/jaxb-tck/tree/master/jaxb-tck/tests/api/jakarta_xml/bind/JAXBContext

-  System property `JAXB_CONTEXT_FACTORY` e.g.

```
export JAXB_CONTEXT_FACTORY=org.eclipse.persistence.jaxb.JAXBContextFactoryService
${JAVA_HOME}/bin/java -jar ${TS_HOME}/lib/javatest.jar -batch -testsuite ${TS_HOME}...
```
or
`${JAVA_HOME}/bin/java -DJAXB_CONTEXT_FACTORY=org.eclipse.persistence.jaxb.JAXBContextFactoryService -jar ${TS_HOME}/lib/javatest.jar -batch -testsuite ${TS_HOME}...`
is ignored. It still try to search JAXB-RI instead of EclipseLink MOXy.

Reply:
to generate `jaxb.properties` one has to use `jaxb-compiler.{sh,bat}` instead of `xjc.{sh,bat}`, see my other note and/or [MOXy source](https://github.com/eclipse-ee4j/eclipselink/blob/master/moxy/org.eclipse.persistence.moxy.utils.xjc/src/main/java/org/eclipse/persistence/jaxb/xjc/MOXyXJC.java#L72)